test/run.sh

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash
```

Comment:
I think it'll be useful to move this to credmark-dev so that it updates on its own. For now, this is good.